src/main/java/net/dv8tion/jda/core/entities/Message.java

```diff
@@ -1254,4 +1263,87 @@ public boolean isImage()
             return height > 0 && width > 0;
         }
     }
+
+    /**
+     * Represents a {@link net.dv8tion.jda.core.entities.Message message} activity like Spotify
+     */
+    class Activity {
+        private final ActivityType type;
+        private final String party_id;
+
+        Activity(ActivityType type, String party_id) {
+            this.type = type;
+            this.party_id = party_id;
+        }
+
+        /**
+         *
+         * @return the type of the activity
+         */
+        public ActivityType getType() {
+            return type;
+        }
```

Comment:
Many of these additions are fairly inconsistent with [JDA's Structure Guide](https://github.com/DV8FromTheWorld/JDA/wiki/6%29-JDA-Structure-Guide)